src/libs/kata-types/src/mount.rs

```diff
@@ -68,6 +68,19 @@ pub fn is_kata_host_dir_volume(ty: &str) -> bool {
     ty == KATA_HOST_DIR_VOLUME_TYPE
 }
 
+/// nydus extraoptions
+#[derive(Debug, serde::Deserialize)]
+pub struct NydusExtraOption {
+    /// source path
+    pub source: String,
+    /// nydus config
+    pub config: String,
+    /// snapshotter directory
+    pub snapshotdir: String,
```

Comment:
I think need a space between snapshot and dir.

Reply:
I think you mean add a underscore? As the `extraoptions` will be deserialized into this struct, but there is no key called `sanpshot_dir` in the `extraotions`.

Reply:
`#[serde(rename(serialize = "xx", deserialize = "yy"))]` may be a choice for different deserialized name and struct member name.

Reply:
Thanks! fixed